src/Native/System.Security.Cryptography.Native/pal_ssl.h

```diff
@@ -1,11 +1,34 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#include "pal_types.h"
+#include "pal_crypto_types.h"
 
 #include <openssl/ssl.h>
 
 /*
+These values should be kept in sync with System.Security.Authentication.SslProtocols.
+*/
+enum SslProtocols : int32_t
+{
+    PAL_SSL_None = 0,
+    PAL_SSL_Ssl2 = 12,
+    PAL_SSL_Ssl3 = 48,
+    PAL_SSL_Tls = 192,
+    PAL_SSL_Tls11 = 768,
+    PAL_SSL_Tls12 = 3072
```

Comment:
I'm guessing these names are PAL_SSL_##managedEnumField, but ALL_CAPS_THEN_Pascal makes for a sad brain in my head.